hs_discover/src/components/Search.vue

```diff
@@ -335,10 +336,22 @@ export default {
     // Allow manual querystring manipulation of freetext search via URL
     if (document.getElementById('qstring').value.trim() !== '') {
       this.searchtext = document.getElementById('qstring').value.trim();
+      this.prefiltered = true;
     }
     // Allow exact subject search via URL particularly for Resource Landing Page
     if (document.getElementById('prefillsubject').value.trim() !== '') {
-      this.subjectFilter = Array(decodeURIComponent(document.getElementById('prefillsubject').value.trim()));
+      const urlEncodedSubj = document.getElementById('prefillsubject').value.trim();
+      const jsSubj = decodeURIComponent(urlEncodedSubj);
+      // Escape backslashes
+      this.subjectFilter = Array(jsSubj);
+      this.prefiltered = true;
+    }
+    if (this.subjectFilter) {
+      const escaped = [];
```

Comment:
wondering why escaped is defined with const rather than let here.

Reply:
method operations are considered a different type of change than assignment, such as x = value in this case, the standard states it should be a const; this is per spec